### PR TITLE
#74 limit maximal resolution of videos

### PR DESCRIPTION
**Pull request by Rpsl**
I tried to solve the problem that I wrote in the issue #74 

I did make refactoring of code about generate arguments for the `youtube-dl` and added new optional param in config.

what do you think?

### CODE REVIEW

File: pkg/ytdl/options_video.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+	options.quality = model.QualityHigh
+
+	if feedConfig.Quality == model.QualityLow {
+		options.quality = model.QualityLow
+	}
+
+	if feedConfig.MaxHeight > 0 {
+		options.maxHeight = feedConfig.MaxHeight
+	}
```
Comment (mxpv) [inline pkg/ytdl/options_video.go]: Why do we need this if checks? why not just:
```go
options.quality = feedConfig.Quality
options.maxHeight = feedConfig.MaxHeight
```

(same for audio)

File: pkg/model/feed.go [hunk 2]
```diff
@@ -17,6 +17,8 @@ const (
 // Format to convert episode when downloading episodes
 type Format string
 
+type MaxHeight int
```
Comment (mxpv) [inline pkg/model/feed.go]: Looks like not used and not needed.

File: pkg/ytdl/options_video.go (hunk 1 above)
Comment (Rpsl) [reply pkg/ytdl/options_video.go]: I wanted to do more validations, but I didn't finish them. You're right, it needs to be cleaned up and brought back to a complete state.

File: pkg/model/feed.go (hunk 2 above)
Comment (Rpsl) [reply pkg/model/feed.go]: fixed

**Pull request merged**